packages/storybook/stories/MeetingComposite/__snapshots__/BasicExample.stories.storyshot

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
For some of these I see a new snapshot file but not a deleted one. I think there might be redundant storyshot files where story filename renames happened.
i.e. `BasicExample.tsx` became `BasicExample.stories.tsx` so jest created a new `BasicExample.stories.storyshot` but did not delete the old `BasicExample.storyshot`

Reply:
will check that

Reply:
storyshot files exist only for .stories files, so no redundancies but I need to remove the ones that are not stories anymore

Reply:
done :)